test/e2e/integration/frontend-test/dynamics.ts

```diff
@@ -1,10 +1,23 @@
 import texts from 'test/e2e/fixtures/texts.json';
+import { changeToLang } from 'test/e2e/integration/frontend-test/formatting';
 import { AppFrontend } from 'test/e2e/pageobjects/app-frontend';
 
 import type { CompOrGroupExternal } from 'src/layout/layout';
 
 const appFrontend = new AppFrontend();
 
+const showReasons = () => {
+  cy.goto('changename');
+  cy.get(appFrontend.changeOfName.newFirstName).type('geir');
+  cy.get(appFrontend.changeOfName.newLastName).type('hoel');
+  cy.get(appFrontend.changeOfName.confirmChangeName)
+    .findByRole('checkbox', {
+      name: /Ja[a-z, ]*/,
+    })
+    .check();
+  cy.get(appFrontend.changeOfName.reasons).should('be.visible');
+};
```

Comment:
I don't see the need to extract this one into a function, unless that function is used multiple places. It just adds one level of indirection.

Reply:
Agree, used it previously, but not needed after all :)